examples/contracts/entry-points-overriding/Cargo.toml

```diff
@@ -13,17 +13,15 @@ crate-type = ["cdylib", "rlib"]
 
 [features]
 library = []
-mt = ["library", "cw-multi-test", "anyhow"]
+mt = ["library", "sylvia/mt", "anyhow"]
 
 [dependencies]
 anyhow = { workspace = true, optional = true }
 cw-multi-test = { workspace = true, optional = true }
```

Comment:
Do we still need this?

Reply:
The `anyhow` and `multitest` deps are removed in the following PR. I missed them here